sdk/e2e/01_create_router_test.py

```diff
@@ -77,9 +77,9 @@ def test_create_router():
     ensembler = DockerRouterEnsemblerConfig(
         image=os.getenv("TEST_ECHO_IMAGE"),
         resource_request=ResourceRequest(
-            min_replica=2, max_replica=2, cpu_request="100m", memory_request="256Mi"
+            min_replica=1, max_replica=1, cpu_request="250m", memory_request="256Mi"
         ),
-        endpoint="anything",
+        endpoint="ensemble",
```

Comment:
Made these changes to the SDK tests so these configs resemble the test configs here: https://github.com/caraml-dev/turing/blob/main/api/e2e/test/testdata/create_router_nop_logger_proprietary_exp.json.tmpl.